### DISCUSSION TITLE
tagging assets

### DISCUSSION THREAD

**Discussion by mareban** [glpi-project/glpi-agent#454] [Ideas]
### Your idea

It would be nice if we can add tags (key and value pair) on asset, like vsphere for example so that tags can be used by automation, reporting, ... ?

ex application owner, business owner, SLA, criticality, ...

**Comment by g-bougard** [comment]
Hi @mareban 
I don't see your idea applied to glpi-agent.
If your idea is for GLPI, you should better talk about it there: https://glpi.userecho.com/
